src/lib/y2network/config.rb

```diff
@@ -140,6 +140,15 @@ def rename_interface(old_name, new_name, mechanism)
       connections.by_interface(old_name).each { |c| c.interface = new_name }
     end
 
+    # deletes interface and all its config. If interface is physical, it is kept.
```

Comment:
You mean that only the configuration is removed, right?

Reply:
yes, we cannot remove physical interface :)

Reply:
hehe, I know :) But I was thinking about some newcomer reading the code (making the distinction between the config and the interface itself).